vividus-plugin-excel/src/main/java/org/vividus/bdd/transformer/ExcelTableTransformer.java

```diff
@@ -94,17 +107,29 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
         return data;
     }
 
+    private List<String> extractValues(IExcelSheetParser sheetParser, String range)
+    {
+        return sheetParser.getDataFromRange(range).stream().map(CellValue::getValue).collect(Collectors.toList());
+    }
+
     private List<String> extractDataFromAddresses(IExcelSheetParser sheetParser, String addresses)
     {
         return Stream.of(addresses.split(";")).map(sheetParser::getDataFromCell).collect(Collectors.toList());
     }
 
-    private String build(List<String> data, TableProperties properties)
+    private String build(List<String> headers, List<List<String>> data, TableProperties properties)
+    {
+        String lineBreakReplacementPropertyValue = properties.getProperties().getProperty("lineBreakReplacement");
+        String lineBreakReplacement = lineBreakReplacementPropertyValue == null ? ""
+            : lineBreakReplacementPropertyValue;
+        List<List<String>> result = data.stream().map(element -> appendSymbols(element, lineBreakReplacement)).
+                                    collect(Collectors.toList());
```

Comment:
please align streams in the following format
```
source.stream()
            .op1()
            .op2()
            .op3()
            ........
```

Reply:
@TatianaTochko ☝️